techniques/failures/F15.html

```diff
@@ -19,32 +32,35 @@
       <section class="procedure"><h3>Procedure</h3>
          <ol>
             <li> Using the accessibility checker for the technology (or if
-                                    that is not available, inspect the code or test with an
+                                    that is not available, inspect the code using a browser's dev tools, or test with an
                                     assistive technology), check the controls to see if they support
-                                    the accessibility API. </li>
+                                    the accessibility API.</li>
          </ol>
       </section>
       <section class="results"><h3>Expected Results</h3>
          <ul>
             <li> If step #1 is false, then this failure condition applies and the
-                                    content fails this Success Criterion </li>
+                                    content fails this success criterion.</li>
          </ul>
       </section>
    </section><section id="related"><h2>Related Techniques</h2></section><section id="resources"><h2>Resources</h2>
       
          <ul>
+           <li>
+             <a href="https://accessibilityinsights.io/docs/windows/overview/">Accessibility Insights for Windows</a>
+           </li>
             <li> 
                   <a href="https://msdn.microsoft.com/en-us/library/ms696082.aspx">Active Accessibility Checker (AccExplorer)</a> 
                </li>
             <li>
```

Comment:
it's no longer being updated because microsoft has transitioned over to UIA rather than msaa/ia2.  but the information there is still useful.  as such, i'd rather we keep it in.   but if someone wants to do a larger review of the links for this doc / others, then maybe that can be done in a different PR.